backend/src/main/java/org/eclipse/tractusx/puris/backend/masterdata/controller/MaterialController.java

```diff
@@ -107,32 +120,29 @@ public ResponseEntity<?> createMaterial(@RequestBody MaterialEntityDto materialD
         @ApiResponse(responseCode = "404", description = "No existing Material Entity found, no update was performed."),
         @ApiResponse(responseCode = "500", description = "Internal Server Error.")
     })
-    public ResponseEntity<?> updateMaterial(@RequestBody MaterialEntityDto materialDto) {
+    public MaterialEntityDto updateMaterial(@RequestBody MaterialEntityDto materialDto) {
         if(!validator.validate(materialDto).isEmpty()) {
             log.warn("Rejected invalid message body.");
-            return ResponseEntity.status(400).build();
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Invalid Material.");
```

Comment:
```suggestion
            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Material is invalid.");
```
Consistency with message below.